content/blog/jamstack-with-pulumi/index.md

```diff
@@ -0,0 +1,359 @@
+---
+title: "Building Jamstack Infrastructure With Pulumi"
+date: 2020-06-09
+meta_desc: "Build infrastructure for deploying a jamstack website with Pulumi."
+meta_image: jamstack.png
+authors:
+    - sophia-parafina
+tags:
+    - jamstack
+    - aws
+---
+
+A Jamstack is a modern architecture for building websites; JAM stands for JavaScript, APIs, and Markup. Jamstacks are deployed on a [CDN](https://en.wikipedia.org/wiki/Content_delivery_network), and content is stored on a cloud services provider. In addition to the speed and simplicity of deploying static content served from a CDN, there are other advantages such as maintaining content with git, modern build tools to generate the static content, automated builds, atomic deploys, and instant cache validation.
+
+While build tools have simplified the process of creating content ready for deployment on a CDN, creating the infrastructure to serve the content remains complicated. . You can use a cloud provider’s web interface or script the build using a CLI tool if you want to manage your infrastructure instead of using a hosted solution. The alternative is to use infrastructure as code tool to automate building and deploying cloud resources. This article demonstrates how to create a jamstack website and deploy it on AWS using Pulumi.
+
+<!--more-->
+
+## Creating Your Website with Hugo
+
+There are many static site generators, but we’ll use hugo to build the site for this example. Instead of installing hugo, we’ll use [Docker](https://docs.docker.com/get-docker/) to run hugo in a [container](https://hub.docker.com/r/klakegg/hugo/). If you have Docker installed, create a directory to hold the website and change into that directory.
```

Comment:
```suggestion
There are many static site generators, but we’ll use [Hugo](https://gohugo.io) to build the site for this example. Instead of installing Hugo, we’ll use [Docker](https://docs.docker.com/get-docker/) to run Hugo in a [container](https://hub.docker.com/r/klakegg/hugo/). If you have Docker installed, create a directory to hold the website and change into that directory.
```